### DISCUSSION TITLE
Now that colima supports XDG dirs, should I version the configs?

### DISCUSSION THREAD

**Discussion by baggiponte** [abiosoft/colima#1093] [Q&A] [answered]
As many, I version my dotfiles under `.config`. Now that colima is XDG compliant on macOS, which configs should I version and which ones should remain private (e.g. add to gitignore)? Thanks :)

**Comment by abiosoft** [comment, marked as answer]
I would say these two. I think everything else are pretty much text files.

```gitignore
# lima directory, includes disk images
_lima/

# unix sockets
*.sock
```

**Reply by dudicoco** [reply]
@abiosoft i'm not sure if it belongs here or in a separate issue, but the `_lima` and the `daemon` dirs should really not live within `.config`, they contain log files, private keys and state, so I believe they belong in other directories.
Please let me know if I should open an issue for this as I believe they should be relocated to the other XDG dirs:
XDG_CACHE_HOME
XDG_DATA_HOME
XDG_STATE_HOME

**Reply by bestlem** [reply]
@dudicoco This should definitely be an issue.


My TM backup deleted old stuff as a 2 G file in ~/.config is not expected. (The _lima directory)

**Reply by abiosoft** [reply]
@dudicoco can you kindly open an issue to track those? I also agree that they should live seperately from the config files.

**Reply by dudicoco** [reply]
https://github.com/abiosoft/colima/issues/1246
